### PR TITLE
Update to Bootstrap 5

### PR DESCRIPTION
**Pull request by RoboticMind**
* Change to the next major bootstrap version
  * Make fixes as needed
  * A few kB saved

* Remove jQuery
  * No longer needed in bootstrap 5 and the only other JS files that use it are simple to change
  * 89.5 kB saved

*  Remove IE11 polyfills
   * IE11 is no longer supported by bootstrap 5 so the polyfills become irrelevant
   * A few kB saved

### CODE REVIEW

File: _layouts/wiki-de.html [hunk 1]
```diff
@@ -36,13 +36,9 @@
...
         <script src="/assets/js/core/jquery-3.6.0.min.js" type="text/javascript"></script>
-        <script src="/assets/js/core/bootstrap-4.6.1.bundle.min.js" type="text/javascript"></script>
-        <script src="/assets/js/details-element-polyfill.js" type="text/javascript"></script>
+        <script src="/assets/js/core/bootstrap-5.1.3.bundle.min.js" type="text/javascript"></script>
         <script> //open dropdown by default if on Desktop, but not on Mobile
-            const breakpoint = parseInt(
-                getComputedStyle(document.documentElement).getPropertyValue('--breakpoint-xl')
-            );
-            if (screen.width >= breakpoint){
+            if (screen.width >= 1200){
```
Comment (barton2526) [inline _layouts/wiki-de.html:41]: is there a specific reason 1200 was selected?

Comment (RoboticMind) [reply _layouts/wiki-de.html:41]: This is the size of the xl breakpoint for bootstrap https://getbootstrap.com/docs/5.0/layout/breakpoints/. I can add a comment there explaining that

Comment (RoboticMind) [reply _layouts/wiki-de.html]: Meant to do that earlier, but forgot about it in the sea of things I had to change

**Comment by RoboticMind** [comment]
I now have this running on my local fork if you want to check for any pages looking strange https://roboticmind.github.io/

**Comment by barton2526** [comment]
> I now have this running on my local fork if you want to check for any pages looking strange https://roboticmind.github.io/

```
Uncaught (in promise) TypeError: document.getElementById(...) is null
    updateDownloads https://roboticmind.github.io/assets/js/fetch-release.js:2
    promise callback* https://roboticmind.github.io/assets/js/fetch-release.js:49
fetch-release.js:2:14
    updateDownloads https://roboticmind.github.io/assets/js/fetch-release.js:2
    (Async: promise callback)
    <anonymous> https://roboticmind.github.io/assets/js/fetch-release.js:49```

**Comment by RoboticMind** [comment]
That shouldn't cause any issues but I can add some code to make the error go away

**Comment by RoboticMind** [comment]
Now the error should be gone and it even saves a request on all the non-index pages as well

**Review by barton2526 — APPROVED** [review]

**Comment by barton2526** [comment]
Anyone else want to review?

**Comment by nathanielcwm** [comment]
Nitpick but the footer links are now underlined and the spacing is different.
.us:
image

Robotic's pages:
image

**Comment by RoboticMind** [comment]
The spacing was changed by bootstrap introducing the `xxl` breakpoint. On the main page, I used some styling to make it close to what it looked like before because the change in width was very noticeable. On other pages, I left it as is because it didn't look significantly different enough to put in the work on all those pages. 

The underlines are in place because bootstrap 5 now keeps all links with underlines (it previously removed all underlines). I could remove the underlines just on the footer, but I kind of like keeping it everywhere.

**Comment by RoboticMind** [comment]
@barton2526 what are your thoughts?

**Pull request merged**